static-instrumenter/agent-instrumenter/src/main/java/io/opentelemetry/contrib/staticinstrumenter/ArchiveEntry.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.staticinstrumenter;
+
+class ArchiveEntry {
+
+  private static final ArchiveEntry NOT_CLASS =
+      new ArchiveEntry("", /* isClass= */ false, /* shouldBeSkipped= */ true);
+
+  private final String className;
+  private final String classPath;
```

Comment:
`classPath` makes me think "classpath" 😄
```suggestion
  private final String path;
```

Reply:
Also consider using `Path` instead of `String`

Reply:
Done. Not using path as later on the path is used as instrumented classes key (and does not contain file extension).